security/fma-holocene.md

```diff
@@ -115,13 +132,13 @@ medium severity / low likelihood
 
 ### Mitigations
 
-The upgrade is designed to function even if the SystemConfig contract does not get updated before the upgrade.  In this case, the system falls back to the Canyon hardcoded parameters, and the system will perform base fee udpates exactly as prior to the upgrade.
+The upgrade is designed to function even if the SystemConfig contract does not get updated before the upgrade. In this case, the system falls back to the Canyon hardcoded parameters, and the system will perform base fee udpates exactly as prior to the upgrade.
 
 We are performing multi-client testing (op-geth and op-reth) and will run a multi-client devnet, so bugs in either will be quickly detected by consensus disagreements among them.
 
-##  L2ToL1MessagePasser Storage Root in Header
+## L2ToL1MessagePasser Storage Root in Header
```

Comment:
```suggestion

```

Reply:
And also remove L21.

Reply:
"L21"?

Reply:
Line 21, it's the table of contents entry for this section. 

No biggie, we can fix this later.